src/packet/packetlist.js

```diff
@@ -134,6 +134,7 @@ class PacketList extends Array {
     packet.packets = packet.packets || new PacketList();
 
     super.push(packet);
+    return this;
```

Comment:
Eventually, I think we should remove this `push` function and just default to the normal `Array.prototype.push` behavior. Since that one returns the new length; I wouldn't `return this` here, as it's inconsistent with that behavior.